systest/mutations_test.go

```diff
@@ -939,7 +1133,7 @@ func EmptyRoomsWithTermIndex(t *testing.T, c *dgo.Dgraph) {
 	require.Equal(t, `{"offices":[{"count(office.room)":1}]}`, string(resp.GetJson()))
 }
 
-func DeleteWithExpandAll(t *testing.T, c *dgo.Dgraph) {
+func (ssuite *SystestTestSuite) Test_M17DeleteWithExpandAll() {
```

Comment:
I am thinking we could move these two tests files into a folder too. It's odd to have them here.